index.html

```diff
@@ -0,0 +1,225 @@
+            <h3 class="main_card_title"> CC Exhibition</h3>
+            <p class="main_card_message">
+              Appreciate various creations applying CC
+              license of artists, organized from Art Center Nabi.
+            </p>
+          </div>
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_forum.svg" alt="forum">
+            <h3 class="main_card_title">Forum</h3>
+            <p class="main_card_message">
+              Have the time to share your thoughts and opinions with experts for each topic more.
+            </p>
+          </div>
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_workshop.svg" alt="workshop">
+            <h3 class="main_card_title">Workshop</h3>
+            <p class="main_card_message">
+              try creating your own work using open source license rather than just watching at it.
+            </p>
+          </div>
+
+          <div class="main_card">
+            <img class="main_card_icon" src="images/icons/icon_ignite.svg" alt="Ignite">
+            <h3 class="main_card_title">Ignite</h3>
+            <p class="main_card_message">
+              Get opportunities to network with CC
+              affiliates around the world, also after the summit
+            </p>
+          </div>
+        </div>
+
+        <button class="join_main_button" type="button">
+          Join CC Global Summit  2023
+        </button>
+
+        <a rel="" href="#" class="link_to_join"> <h4> SEE THE WHOLE PROGRAM </h4></a>
+
+      </div>
+    </section>
+      <!-- End of main section  -->
+      <!-- feature speaker start  -->
+    <section class="featured_speakers_section" id="speakers_section">
+      <h3 class="featured_speakers_header_text">
+        Feature speakers
+      </h3>
+      <div class="featured_speakers_text_underline"></div>
+
+      <div class="featured_speakers_main_div" id="featured_speakers">
+        <!--speakers-->
+      </div>
+      <div class="expand_more_place_holder">
+        <span class="more_text">
+          MORE
+          <span><img class="expand_arrow" src="images/icons/icons_expand_arrow.png" alt="expand arrow"></span>
+        </span>
+
+      </div>
+    </section>
+    <!-- feature speaker end  -->
+    <!-- partner section start  -->
+    <section class="partners_part" id="partners">
+      <h3 class="partners_part_title">Partner</h3>
+      <div class="partners_part_title_style"></div>
+      <div class="partners_logos">
+        <img class="partners_logo_image_style" src="images/logo/amazon.png" alt="grovara">
+        <img class="partners_logo_image_style" src="images/logo/airbnb.png" alt="kamara yokie">
+        <img class="partners_logo_image_style" src="images/logo/deezer.png" alt="Scorpion Garage">
+        <img class="partners_logo_image_style" src="images/logo/linkedin.png" alt="Scorpion Garage">
+      </div>
+    </section>
```

Comment:
- [ ] Could you please align the brand of your partners to match the template? Currently, they're not aligned correctly. 😊

   ### Your design:
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/76c0b752-9cb5-4216-9e9b-fd7f4d9173d2)


   ### Behance design:
   ![image](https://github.com/Nelofarzabi/Capstone_Project/assets/107952724/4be75c3b-665f-4f79-af04-0a26c7894ff8)